### PR TITLE
Fixed window not ignoring clicks in xfce

### PR DESCRIPTION
**Pull request by teschnei**

### CODE REVIEW

File: cef/hudkit_window.cc [hunk 1]
```diff
@@ -52,6 +52,7 @@ void HudkitWindow::Initialize()
         gtk_window_set_accept_focus(gtkWindow, false);
         gtk_window_set_decorated(gtkWindow, false);
         gtk_window_set_resizable(gtkWindow, false);
+        lockedState = true;
```
Comment (valarnin) [inline cef/hudkit_window.cc:55]: Was this an intentional addition?

Comment (valarnin) [inline cef/hudkit_window.cc:55]: Was this an intentional addition?

Comment (teschnei) [reply cef/hudkit_window.cc:55]: Yeah, this block (not used currently) locks the window on creation, so lockedState should be true if we start it locked

**Comment by valarnin** [comment]
I'll probably hold off on creating a new CEF build for this, since there are other issues that still need resolved. Going to see if I can get some time this weekend to work on this project a bit.

**Pull request merged**